draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,494 @@
+submitted by other funded parties, and satisfy similar levels of public 
+scrutiny.
+
+All substantial software whose development was funded by the Dev Fund SHOULD 
+be released under an Open Source license (as defined by the Open Source 
+Initiative [#osd]_), preferably the MIT license.
+
+The ZF SHALL continue to operate the Zcash Community Advisory Panel and SHOULD 
+work toward making it more representative and independent (more on that below).
+
+Enforcement
+~~~~~~~~~~~
+
+For grant recipients, these conditions SHOULD be included in their contract 
+with FPF, such that substantial violation, not promptly remedied, will cause 
+forfeiture of their grant funds and their return to FPF.
+
+BP, ECC, ZF and FPF MUST contractually commit to each other to fulfill these 
+conditions, and the prescribed use of funds, such that substantial violation, 
+not promptly remedied, will permit the other parties to issue a modified 
+version of Zcash node software that removes the violating party's Dev Fund 
+slice, and use the Zcash trademark for this modified version. The slice's funds 
+will be reassigned to ZCG (whose integrity is legally protected by the 
+Restricted Fund treatment).
+
+
+Amendments and Replacement of the Dev Fund
+------------------------------------------
+
+Nothing in this ZIP is intended to preclude any amendments to the Dev Fund 
+(including but not limited to, changes to the Dev Fund allocation and/or the 
+addition of new Dev Fund recipients), if such amendments enjoy the consensus 
+support of the Zcash community. 
+
+Nothing in this ZIP is intended to preclude replacement of the Dev Fund with a 
+different mechanism for ecosystem development funding. 
+
+BP, ECC, ZF and FPF SHOULD facilitate the amendment or replacement of the Dev 
+Fund if there is sufficient community support for doing so. 
+
+
+Future Community Governance
+---------------------------
+
+Decentralized community governance is used in this proposal via the Zcash 
+Community Advisory Panel as input into the Zcash Community Grants Committee 
+which governs the `ZCG slice (Zcash Community Grants)`_.
+
+It is highly desirable to develop robust means of decentralized community
+voting and governance, either by expanding the Zcash Community Advisory Panel 
+or a successor mechanism. BP, ECC, FPF and ZCG SHOULD place high priority on 
+such development and its deployment, in their activities and grant selection.
+
+
+ZF Board Composition
+--------------------
+
+Members of ZF's Board of Directors MUST NOT hold equity in ECC or have current 
+business or employment relationships with ECC.
+
+The Zcash Foundation SHOULD endeavor to use the Zcash Community Advisory Panel 
+(or successor mechanism) as advisory input for future board elections.
+
+
+FPF Board Composition
+--------------------
+
+Members of FPF's Board of Directors MUST NOT hold equity in ECC or have current 
+business or employment relationships with ECC. 
```

Comment:
I guess it could be removed, although (in theory) Bootstrap could sell some or all of ECC's equity to a third party. I'll amend to add reference to Bootstrap for now, pending further feedback.